### PR TITLE
Main-sect

### PR DESCRIPTION
**Pull request by NataDav-create**

### CODE REVIEW

File: content/home.js [hunk 1]
```diff
@@ -11,8 +11,69 @@ module.exports = {
     title: `Gold Demands <br/> 2022`,
     subtitle: 'Subtitle',
   },
+  introArticle: {
+    title: 'Key insight 3',
```
Comment (a-guris) [inline content/home.js]: are you sure this title goes here?

File: content/home.js [hunk 2]
```diff
@@ -11,8 +11,69 @@ module.exports = {
...
+    text:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Bibendum est ultricies integer quis. Iaculis urna id volutpat lacus laoreet. Mauris vitae ultricies leo integer malesuada. Ac odio tempor orci dapibus ultrices in. Egestas diam in arcu cursus euismod.\n' +
+      'Dictum fusce ut placerat orci nulla. Tincidunt ornare massa eget egestas purus viverra accumsan in nisl. Tempor id eu nisl nunc mi ipsum faucibus. Fusce id velit ut tortor pretium. Massa ultricies mi quis hendrerit dolor magna eget.',
+  },
+  articleOne: {
```
Comment (a-guris) [inline content/home.js]: article might not be the correct name for this, as the whole page is basically an article

File: content/home.js [hunk 3]
```diff
@@ -11,8 +11,69 @@ module.exports = {
...
+  articleFour: {
+    title: 'Key insight 4',
+    text: 'Dui faucibus in ornare quam. In iaculis nunc sed augue lacus viverra vitae congue. Vitae tempus quam pellentesque nec nam aliquam sem et. Ut morbi tincidunt augue interdum. Sem fringilla ut morbi tincidunt augue. Morbi enim nunc faucibus a pellentesque sit amet porttitor eget. In est ante in nibh mauris. Nam aliquam sem et tortor consequat id porta nibh. Diam quis enim lobortis scelerisque fermentum dui faucibus. Non curabitur',
+    conclusion:
+      'Learn more in the full report, lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. ',
+  },
+  quoteOne: {
+    text: 'Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores.',
+    authorTitle: `Author name <br/> Designation, Company name`,
+    quoteImg: './images/articles/quotes.png',
```
Comment (a-guris) [inline content/home.js]: remove period before first slash

File: content/home.js [hunk 4]
```diff
@@ -11,8 +11,69 @@ module.exports = {
...
+    prevMobile: '/images/common/prevMobile.png',
+    mobileText: 'Prev',
+  },
+  navigationNext: {
+    navigationImage: '/images/navigation/navigation2.jpg',
+    title: 'Gold Market Commentary',
+    text: 'Lorem ipsum dolor sit amet, consec tetur adipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et.',
+    direction: 'Next',
+    nextMobile: '/images/common/nextMobile.png',
+    mobileText: 'Next',
```
Comment (a-guris) [inline content/home.js]: the text for both mobile and desktop direction buttons (next, prev) are universal, they can be moved into the component itself

File: src/components/Hero/Hero.style.js [hunk 5]
```diff
@@ -7,7 +7,7 @@ import { p2r, SecondaryProps } from '../../styles/fonts';
 const styles = (image) => css`
   background-color: ${colors.bgWhite};
   width: 100%;
-  height: 100%;
+  //height: 100%;
```
Comment (a-guris) [inline src/components/Hero/Hero.style.js]: remove comments

File: src/components/Layout/Layout.jsx [hunk 6]
```diff
@@ -5,7 +5,7 @@ import UtilWebVitals from '@economist/fabric-components/utility/utilwebvitals/Ut
 import React, { useEffect } from 'react';
 
 import { appendAdditionalScripts } from '../../utils/utils/cookiesConsent';
-import { MobileMenuBtn } from '../MobileMenuBtn';
+// import { MobileMenuBtn } from '../MobileMenuBtn';
```
Comment (a-guris) [inline src/components/Layout/Layout.jsx]: remove comments

File: src/components/Layout/Layout.jsx [hunk 7]
```diff
@@ -38,7 +38,7 @@ const Layout = ({ children, title, description, image }) => {
       <Seo title={title} description={description} image={image} /> <PreloadStatic />
       <div css={styles}>
         <div className="container">
-          <MobileMenuBtn />
+          {/* <MobileMenuBtn /> */}
```
Comment (a-guris) [inline src/components/Layout/Layout.jsx]: remove comments

**Review by a-guris — APPROVED** [review]

File: content/home.js (hunk 1 above)
Comment (NataDav-create) [reply content/home.js]: No, forgot to delete

File: src/components/Hero/Hero.style.js (hunk 5 above)
Comment (NataDav-create) [reply src/components/Hero/Hero.style.js]: Done

File: src/components/Layout/Layout.jsx (hunk 6 above)
Comment (NataDav-create) [reply src/components/Layout/Layout.jsx]: Done

File: src/components/Layout/Layout.jsx (hunk 7 above)
Comment (NataDav-create) [reply src/components/Layout/Layout.jsx]: Done

File: content/home.js (hunk 3 above)
Comment (NataDav-create) [reply content/home.js]: Done

**Review by a-guris — APPROVED** [review]

**Pull request merged**